### PR TITLE
[minor] Extend 'suite_manage_load_dbc_scripts' role to support health standalone

### PR DESCRIPTION
**Pull request by MollyGu-CN**
Add a variable MAS_APP_ID to support health standalone case.

Tested in one of the health standalone env

image

image

### CODE REVIEW

File: ibm/mas_devops/roles/suite_manage_load_dbc_scripts/tasks/main.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@
       debug:
         msg:
           - "Instance ID ............... {{ mas_instance_id }}"
+          - "App ID ............... {{ mas_app_id }}"
```
Comment (andrercm) [inline ibm/mas_devops/roles/suite_manage_load_dbc_scripts/tasks/main.yaml:52]: Would you mind fixing the standard dots in the debug just to keep it consistent with other lines?

File: ibm/mas_devops/roles/suite_manage_load_dbc_scripts/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (andrercm) [inline ibm/mas_devops/roles/suite_manage_load_dbc_scripts/defaults/main.yml:1]: would you be able to add an extra interna var, something as:
```
mas_app_id_supported:
  - 'manage'
  - 'health'
```

and then add an assertion right at the beginning of the `tasks/main.yml` that validates that the passed `mas_app_id` matches one of the `mas_app_id_supported`?

Here's a sample of how you'd do it:
https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d/tasks/main.yml#L13

File: ibm/mas_devops/roles/suite_manage_load_dbc_scripts/vars/main.yaml [hunk 3]
```diff
@@ -1,7 +1,9 @@
 ---
```
Comment (andrercm) [inline ibm/mas_devops/roles/suite_manage_load_dbc_scripts/vars/main.yaml:1]: It doesn't seem good practice to have default vars in `vars/main.yaml` file... not sure why this was done like this in first place... instead we should get rid of this folder/file and move all the default vars to the new `defaults/main.yml` you created.

**Review by andrercm — CHANGES_REQUESTED** [review]
Can you work on the couple of changes ?

File: ibm/mas_devops/roles/suite_manage_load_dbc_scripts/tasks/main.yaml (hunk 1 above)
Comment (MollyGu-CN) [reply ibm/mas_devops/roles/suite_manage_load_dbc_scripts/tasks/main.yaml]: Added

File: ibm/mas_devops/roles/suite_manage_load_dbc_scripts/defaults/main.yml (hunk 2 above)
Comment (MollyGu-CN) [reply ibm/mas_devops/roles/suite_manage_load_dbc_scripts/defaults/main.yml]: Added and tested
image

File: ibm/mas_devops/roles/suite_manage_load_dbc_scripts/vars/main.yaml (hunk 3 above)
Comment (MollyGu-CN) [reply ibm/mas_devops/roles/suite_manage_load_dbc_scripts/vars/main.yaml:1]: Moved

Comment (MollyGu-CN) [reply ibm/mas_devops/roles/suite_manage_load_dbc_scripts/vars/main.yaml:1]: vars/main.yaml is deleted, but the folder is still there? I have removed entire folder in local, but it doesn't appear in the commit...

**Review by andrercm — APPROVED** [review]
Thanks for making the changes

**Pull request merged**